res/values/strings.xml

```diff
@@ -629,33 +629,33 @@
     <string name="name_desktop">Name</string>
     <string name="autocrypt_key_transfer_desktop">Autocrypt key transfer</string>
     <string name="initiate_key_transfer_title_desktop">Initiate key transfer</string>
-    <string name="initiate_key_transfer_desktop">An Autocrypt Setup Message securely shares your end-to-end setup with other Autocrypt-compliant apps. The setup will be encrypted by a setup code which is displayed here and must be typed on the other device.</string>
+    <string name="initiate_key_transfer_desktop">An Autocrypt Setup Message securely shares your end-to-end setup with other Autocrypt-compliant apps. The setup will be encrypted by a setup code displayed here, to be typed on the other device.</string>
     <string name="reply_to_message_desktop">Reply to message</string>
     <string name="select_group_image_desktop">Select group image</string>
     <string name="imex_progress_title_desktop">Backup progress</string>
     <string name="download_attachment_desktop">Download attachment</string>
     <string name="export_backup_desktop">Export backup</string>
     <string name="transfer_key_desktop">Transfer key</string>
-    <string name="show_key_transfer_message_desktop">Your key has been sent to yourself. Switch to the other device and open the setup message. You should be prompted for a setup code. Type the following digits into the prompt. Once you\'re done, your other device will be ready to use Autocrypt.</string>
+    <string name="show_key_transfer_message_desktop">Your key has been sent to yourself. Switch to the other device and open the Autocrypt Setup Message. You should be prompted for a setup code. Once the following digits are typed into your other device, it will be ready to use Autocrypt.</string>
```

Comment:
```suggestion
    <string name="show_key_transfer_message_desktop">Your key has been sent to yourself. Switch to the other device and open the Autocrypt Setup Message. You should be prompted for a setup code. Once the following digits are typed into on other device, it will be ready to use Autocrypt.</string>
```